src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect2.java

```diff
@@ -64,29 +69,35 @@
     InnerInputJavadocParagraphIncorrect anon = new InnerInputJavadocParagraphIncorrect() {
 
             /**
-         * <p>Some Javadoc. // violation
+         * <p>Some Javadoc. // violation 'Javadoc content should start from the next line after
+         * /\*\*.'
          *
-         * Some Javadoc. // violation above
+         * Some Javadoc. // violation above 'Javadoc content should start from the next line after
+         * /\*\*.'
```

Comment:
It is difficult to read the message after manually escaping every character, please use `\Q` and `\E`.

Reply:
Changes Done

Reply:
Only use `\Q \E` where you need to escape many regex characters, for example if some violation message is like-
```
variable name must match the pattern ^[a-zA-Z_$][a-zA-Z_$0-9]*$
```
you can see at the end there are lot of regex characters, escaping them manually would make the violation message difficult to read and would also create a lot of noise in the violation message, so to overcome this problem you can simply change it to-
```
variable name must match the pattern \Q ^[a-zA-Z_$][a-zA-Z_$0-9]*$ \E
```
Don't use `\Q \E` when there are no special regex characters, for eg-
```
tag should be placed immediately before the first word, with no space after
```
There are no characters that need to be escaped so usage of `\Q \E` is unnecessary, please make these changes in all relevant cases.

Reply:
Thanks a lot @Vyom-Yadav for the easy explaination.